src/runtime-rs/crates/runtimes/virt_container/src/sandbox.rs

```diff
@@ -68,19 +71,28 @@ impl VirtSandbox {
             agent,
             hypervisor,
             resource_manager,
-            monitor: Arc::new(HealthCheck::new(true, true)),
+            monitor: Arc::new(HealthCheck::new(true, false)),
         })
     }
 
     async fn prepare_for_start_sandbox(
         &self,
+        _id: &str,
         netns: Option<String>,
-        _config: &TomlConfig,
+        config: &TomlConfig,
     ) -> Result<Vec<ResourceConfig>> {
         let mut resource_configs = vec![];
 
-        if let Some(_netns_path) = netns {
-            // TODO: support network
+        if let Some(netns_path) = netns {
+            let network_config = ResourceConfig::Network(NetworkConfig::NetworkResourceWithNetNs(
+                NetworkWithNetNsConfig {
+                    network_model: config.runtime.internetworking_model.clone(),
+                    netns_path,
+                    // TODO: get queues from config
```

Comment:
From the `TomlConfig` specified to this function?

Reply:
Thanks for your review, update to get from config.